index.html

```diff
@@ -73,5 +61,138 @@ <h3 class="h3_title">I’am a Software Developer</h3>
+          <li><button class="butn_M">bootstrap</button></li>
+          <li><button class="butn_M">Ruby</button></li>
+        </ul>
+        <button class="btn_M_en">See project</button>
+      </section>
+      <section class="section_M">
+        <h2 class="h1_3">Profesional Art Printing Data</h2>
+        <p class="p_sectionM">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ul_sectionM">
+          <li><button class="butn_M">html</button></li>
+          <li><button class="butn_M">bootstrap</button></li>
+          <li><button class="butn_M">Ruby</button></li>
+        </ul>
+        <button class="btn_M_en">See project</button>
+      </section>
+      <section class="section_M">
+        <h2 class="h1_3">Profesional Art Printing Data</h2>
+        <p class="p_sectionM">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ul_sectionM">
+          <li><button class="butn_M">html</button></li>
+          <li><button class="butn_M">bootstrap</button></li>
+          <li><button class="butn_M">Ruby</button></li>
+        </ul>
+        <button class="btn_M_en">See project</button>
+      </section>
+      <section class="section_M">
+        <h2 class="h1_3">Profesional Art Printing Data</h2>
+        <p class="p_sectionM">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ul_sectionM">
+          <li><button class="butn_M">html</button></li>
+          <li><button class="butn_M">bootstrap</button></li>
+          <li><button class="butn_M">Ruby</button></li>
+        </ul>
+        <button class="btn_M_en">See project</button>
+      </section>
+      <section class="section_M">
+        <h2 class="h1_3">Profesional Art Printing Data</h2>
+        <p class="p_sectionM">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ul_sectionM">
+          <li><button class="butn_M">html</button></li>
+          <li><button class="butn_M">bootstrap</button></li>
+          <li><button class="butn_M">Ruby</button></li>
+        </ul>
+        <button class="btn_M_en">See project</button>
+      </section>
+    </section>
+
+    <section class="section_aboutMe">
+      <h2 class="h2_4">About me</h2>
+      <p class="p_4">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <button class="btn_about">Get My Resume</button>
```

Comment:
- [ ] Please remove the borderline to match with the  Figma design. You can use `border: none in the CSS file to remove the border.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![y1](https://user-images.githubusercontent.com/70483835/159112124-3f695024-5628-4854-b0ac-09fb53d225c5.png) | ![f1](https://user-images.githubusercontent.com/70483835/159112088-b18eb765-a272-44a1-9221-3967ec12ce35.png)